game.js

```diff
@@ -0,0 +1,48 @@
+const COMP_WINS = 0;
+const USER_WINS = 1;
+
+const computerSelection = computerPlay();
+console.log(computerSelection);
+const buttons = document.querySelector(".js-buttons-container"); //selected parent container
+buttons.addEventListener("click", choosePlayer); // adds listener on each clicked button
```

Comment:
The event listener function is usually defined inline if it's not too long. E.g. `buttons.addEventListener("click", function(e) { ... });`